pkg/cluster/install.go

```diff
@@ -343,6 +343,7 @@ func (m *manager) Install(ctx context.Context) error {
 			steps.Action(m.initializeOperatorDeployer), // depends on kube clients
 			steps.Action(m.removeBootstrap),
 			steps.Action(m.removeBootstrapIgnition),
+			steps.Condition(m.apiServersReady, 30*time.Minute, true), // waiting for API Server to become ready
```

Comment:
Should we put a comment in here stating the apiserver will occasionally be disrupted and the step to configure certificates will fail?  Currently under investigation.

Reply:
Comment updated